src/Components/List/List.css

```diff
@@ -85,3 +85,34 @@
 .resetButton:hover {
   cursor: pointer;
 }
+
+/* Issue #12 css */
+.inactive {
+  background-color: rgb(165, 164, 164);
+  padding: 0.5rem 1rem;
```

Comment:
We also noticed that you have multiple CSS rules for the border-radius and the padding. I think we can consolidate this by adding a more global class or targeting the label itself for these rules.

Reply:
Noted and updated! 😄